cmd/oceanbench/s/broadcast.js

```diff
@@ -8,35 +8,31 @@ document.addEventListener('DOMContentLoaded', function() {
   if (startTimeUnix) sync('start-time', 'start-time-unix', 'time-zone', false);
   if (endTimeUnix) sync('end-time', 'end-time-unix', 'time-zone', false);
 
-  sensorList.forEach(sensor => {
-    if (sensor.SendMsg) {
-      document.getElementById(sensor.Name).checked = true;
-    }
-  });
+  if (sensorList){
+      sensorList.forEach(sensor => {
+      if (sensor.SendMsg) {
+        document.getElementById(sensor.Name).checked = true;
+      }
+    });
+  }
 
   if (sendMsg) {
     document.getElementById("report-sensor").checked = true;
   }
 
-  document.getElementById("header").addEventListener("site-change", handleSiteChange)
+  document.getElementById("header").addEventListener("site-change", handleSiteChange);
 });
 
 function handleSiteChange(event) {
-  // Check that the user really wants to change the site.
-  let resp = confirm("Unsaved changes may be lost!\nAre you sure you want to change sites?")
-  if (resp) {
-    location.assign("/admin/broadcast"); // This will empty the form.
-  } else {
-    // make a request to keep the old site.
```

Comment:
Yeah that's right. I think for now it's fine to assume the user knows they can lose data if they don't click save and change the site menu. It would be nice to warn if not saved in the future but not needed.